src/main/java/com/example/demo/repository/DateMapper.java

```diff
@@ -10,24 +10,20 @@
 @Mapper
 public interface DateMapper {
 
-	    //全件取得
-		public List<DateCalc> findAll();
-		
-		//1件取得
-		public DateCalc findById(DateCalc dateCalc);
+	// 全件取得
+	public List<DateCalc> findAll();
 
-		// 登録
-		public void insertOne(DateCalc date);
+	// 1件取得
+	public DateCalc findById(int id);
```

Comment:
存在しない場合を考慮して以下としたいです。
```suggestion
	public Optional<DateCalc> findById(int id);
```